proposals/lip-0050.md

```diff
@@ -239,24 +338,21 @@ def execute(trs: Transaction) -> None:
     legacyAddress = getLegacyAddress(trs.senderPublicKey)
     delete legacyAccounts(legacyAddress) from the legacy accounts substore
 
-    newAddress = 20-byte address derived from trs.senderPublicKey
+    newAddress = SHA-256(trs.senderPublicKey)[:NUM_BYTES_ADDRESS]
     # Unlock the tokens in the legacy reserve account and transfer them to the
     # account reclaiming the tokens.
-    token.unlock(ADDRESS_LEGACY_RESERVE, MODULE_ID_LEGACY, TOKEN_ID_LSK_MAINCHAIN, trs.params.amount)
+    token.unlock(ADDRESS_LEGACY_RESERVE, MODULE_NAME_LEGACY, TOKEN_ID_LSK, trs.params.amount)
```

Comment:
I think that new convention says that we capitalize the module name.
```suggestion
    Token.unlock(ADDRESS_LEGACY_RESERVE, MODULE_NAME_LEGACY, TOKEN_ID_LSK, trs.params.amount)
```

Also below

Reply:
Changed everywhere.